extra_patches/0001-build_proof-test-vectors.patch

```diff
@@ -0,0 +1,82 @@
+
+ .../eth2spec/test/altair/unittests/test_helpers.py | 11 ++++++++---
+ tests/generators/merkle/main.py                    | 14 ++++++++++++++
+ tests/generators/merkle/requirements.txt           |  2 ++
+ 3 files changed, 24 insertions(+), 3 deletions(-)
+ create mode 100644 tests/generators/merkle/main.py
+ create mode 100644 tests/generators/merkle/requirements.txt
+
+diff --git a/tests/core/pyspec/eth2spec/test/altair/unittests/test_helpers.py b/tests/core/pyspec/eth2spec/test/altair/unittests/test_helpers.py
+index c837f06c..f0a8166a 100644
+--- a/tests/core/pyspec/eth2spec/test/altair/unittests/test_helpers.py
++++ b/tests/core/pyspec/eth2spec/test/altair/unittests/test_helpers.py
+@@ -9,10 +9,11 @@ from eth2spec.test.helpers.merkle import build_proof
+ @with_phases([ALTAIR])
+ @spec_state_test
+ def test_next_sync_committee_tree(spec, state):
+-    state.next_sync_committee: object = spec.SyncCommittee(
+-        pubkeys=[state.validators[i]for i in range(spec.SYNC_COMMITTEE_SIZE)]
+-    )
++    yield "state", state
++    yield "leaf", state.next_sync_committee.hash_tree_root()
++    yield "index", "meta", spec.NEXT_SYNC_COMMITTEE_INDEX
+     next_sync_committee_branch = build_proof(state.get_backing(), spec.NEXT_SYNC_COMMITTEE_INDEX)
++    yield "proof", next_sync_committee_branch
+     assert spec.is_valid_merkle_branch(
+         leaf=state.next_sync_committee.hash_tree_root(),
+         branch=next_sync_committee_branch,
+@@ -25,7 +26,11 @@ def test_next_sync_committee_tree(spec, state):
+ @with_phases([ALTAIR])
+ @spec_state_test
+ def test_finality_root_tree(spec, state):
++    yield "state", state
++    yield "leaf", state.finalized_checkpoint.root
++    yield "index", "meta", spec.FINALIZED_ROOT_INDEX
+     finality_branch = build_proof(state.get_backing(), spec.FINALIZED_ROOT_INDEX)
++    yield "proof", finality_branch
+     assert spec.is_valid_merkle_branch(
+         leaf=state.finalized_checkpoint.root,
+         branch=finality_branch,
+diff --git a/tests/generators/merkle/main.py b/tests/generators/merkle/main.py
+new file mode 100644
+index 00000000..53cb59de
+--- /dev/null
++++ b/tests/generators/merkle/main.py
+@@ -0,0 +1,14 @@
++from eth2spec.test.helpers.constants import ALTAIR
++from eth2spec.gen_helpers.gen_from_tests.gen import run_state_test_generators
++
++
++if __name__ == "__main__":
++    altair_mods = {key: 'eth2spec.test.altair.unittests.test_' + key for key in [
++        'helpers',
++    ]}
++
++    all_mods = {
++        ALTAIR: altair_mods
++    }
++
++    run_state_test_generators(runner_name="merkle", all_mods=all_mods)
+diff --git a/tests/generators/merkle/requirements.txt b/tests/generators/merkle/requirements.txt
+new file mode 100644
+index 00000000..18224868
+--- /dev/null
++++ b/tests/generators/merkle/requirements.txt
+@@ -0,0 +1,2 @@
++pytest>=4.4
++../../../[generator]
+-- 
+2.30.1 (Apple Git-130)
```

Comment:
It seems odd to add the git version information here?

Reply:
That's just the `git format-patch` default.

```
--[no-]signature=<signature>
Add a signature to each message produced. Per RFC 3676 the signature is separated from the body by a line with '-- ' on it. If the signature option is omitted the signature defaults to the Git version number.
```

Will remove it.